src/StaticCaching/Cachers/Writer.php

```diff
@@ -17,7 +17,7 @@ class Writer
      */
     public function write($path, $content, $lockFor = 0)
     {
-        @mkdir(dirname($path), 0777, true);
+        @mkdir(dirname($path), config('statamic.static_caching.permissions.directory', 0755), true);
```

Comment:
Should the defaults not be the same as what they were before (`0777`), in case changing the default causes issues for anyone? I can't imagine it would but 🤷‍♂️

Reply:
I did have that then changed them to be the ones Jason was changing them to in his PR (which seem better to me than 777!)

Reply:
Oh okay, ignore me then!